### PR TITLE
User model

### PR DESCRIPTION
**Pull request by sumaerjolly**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,12 @@
 Rails.application.routes.draw do
+  get 'users/new'
```
Comment (dipto0321) [inline config/routes.rb]: No need.

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/sign_up', to: 'users#new'
+  get    '/sign_in',   to: 'sessions#new'
+  post   '/sign_in',   to: 'sessions#create'
+  delete '/sign_out',  to: 'sessions#destroy'
+  resources :sessions, only: [:new, :create, :destroy]
```
Comment (dipto0321) [inline config/routes.rb]: No need.

File: app/models/post.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class Post < ApplicationRecord
```
Comment (dipto0321) [inline app/models/post.rb:1]: Let's use validation for post title and body. So that user can't submit any empty post.

**Review by dipto0321 — CHANGES_REQUESTED** [review]
Hi @sumaerjolly ,

Good job 👏 . Please check those comment on code. This is a collaborative project.



Please contact with student success if you have any issues with your partner. Here is an article about [this](https://microverse.zendesk.com/hc/en-us/articles/360026622993-I-did-a-collaborative-project-solo-What-happens-now-). Please add a descriptive readme about this projects, user rubocop to fix all the linter related issues and add stickler in this project.

After complete those ☝️ jobs send another request to re-review this project. 

Thanks 🙏

**Comment by azdrenymeri** [comment]
### Hello @sumaerjolly :wave:
#### Nice job on this project. :+1:
#### There are no further requests, you can go ahead and merge it.

**Pull request merged**